_author_instructions/05_tutorials.md

```diff
@@ -8,6 +8,7 @@ permalink: /authors/tutorials/
 ---
 
 ## What is a publishable tutorial and/or training article?
+As detailed below, a tutorial is largely a procedural guide to a particular piece of software, whereas a training article is more introdcutory and provides general background preparation for diverse software packages.
```

Comment:
Suggest "provides an introduction to a procedure or theory and thus provides" rather than "is more introductory and provides", since it's not clear-cut that one is more introductory than another.   Also suggest "general background preparation for performing molecular simulation calculations with diverse software packages", to clarify what people would do with the software packages.